third-party/realdds/doc/DDS Use cases.md

```diff
@@ -0,0 +1,255 @@
+
+realdds is a library to help control a realsense camera over network using librealsense API.
```

Comment:
"to help control" is weird, because we understand from that that we can get help from this library, or do it in another way, while we do want this to be the only way, right?
So I would have chosen: "realdds is a library that enables to control..." or that permits the user to control. Or something like that.

Reply:
I will rephrase

Our design is such that
1. realdds is not limited to LibRS cameras (and does not use any librealsense types)
2. It is possible to control cameras without realdds, but it provides "off the shelf" types and helper functions, that make it easier and faster to write such control applications.

Reply:
Changed the first sentence to
> realdds is a library that ease the control of a realsense camera over network using librealsense API.
realdds provides "off the shelf" types and logic that makes it easier and faster to write applications that control a camera over the network.

Reply:
ok